test-graal-native-image/pom.xml

```diff
@@ -118,7 +137,7 @@
           <plugin>
             <groupId>org.graalvm.buildtools</groupId>
             <artifactId>native-maven-plugin</artifactId>
-            <version>0.10.5</version>
+            <version>0.10.6</version>
```

Comment:
Increasing the version was not strictly necessary, and is not directly related to this issue.

Reply:
LGTM